project/tests/test_schema.py

```diff
@@ -32,6 +32,26 @@ def test_logout_works(graphql_client):
     assert graphql_client.request.user.pk is None
 
 
+class TestClearAnonymousSession:
+    QUERY = "mutation { output: clearAnonymousSession(input: {}) { session { csrfToken } } }"
+
+    def test_it_does_nothing_if_user_is_logged_in(self, db, graphql_client):
+        user = UserFactory()
+        graphql_client.request.user = user
+        graphql_client.request.session["boop"] = 1
+        result = graphql_client.execute(self.QUERY)
+        assert len(result["data"]["output"]["session"]["csrfToken"]) > 0
```

Comment:
Is the goal that the CSRF token changes if the user's session is reset, but stays the same if the user's session stays the same? If so, maybe we should check that the values are the same before and after the request.